lib/src/components/NxCheckbox/NxCheckbox.scss

```diff
@@ -36,7 +36,6 @@ $-width: 16px;
   + .nx-checkbox__box {
     background-color:var(--nx-color-interactive-background-disabled);
     border-color: var(--nx-color-interactive-border-disabled);
-
```

Comment:
Extraneous and undesirable change

Reply:
> Extraneous and undesirable change

Ops, totally missed this. Fixed:
5c5ea6780bb688265db64fa897b7852cd8c5935e